File: src/main/java/com/example/ex3_2_back/entity/Actor.java
package com.example.ex3_2_back.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Builder
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Actor {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id;
    @Builder.Default
    Gender gender = Gender.Unknown;
    String name;
}

File: src/main/kotlin/com/example/ex3_2_back/entity/Actor.kt
package com.example.ex3_2_back.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id

@Entity
class Actor(
    @Id
    @Column(name = "actor_id")
    var id: Int = 0,
    var gender: Gender = Gender.Unknown
)